frontend/src/main.js

```diff
@@ -56,6 +55,8 @@ function initApp() {
   library.add(faSpinner)
   library.add(faTimes)
   library.add(faQuestionCircle)
+  library.add(faPlus)
```

Comment:
I don't think we are using faPlus. To improve performance and minimize size we don't want to include anything that's unused.